tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -870,9 +945,7 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
                 genNonNativeJSClassConstructor(className),
                 Nil)
           } else {
-            js.Apply(
-                js.New(encodeClassVar(className), Nil) DOT js.Ident("init___"),
-                Nil)
+            js.New(encodeClassVar(className), Nil)
```

Comment:
This implicitly uses the fact that every module class only has a single init method, right? I'm fine with imposing this, but is it IR checked? (Before, one could theoretically add another constructor and instantiate the class by hand, not that it is a good idea).